js/objectsBasics.js

```diff
@@ -0,0 +1,550 @@
+/*
+OBJECTS
+    An object is a collection of properties. A property is an 
+    association between a name (or key) and a value.
+    An object is created using the curly braces {...}.
+    An empty object can be created using the two ways:
+*/
+    // 1.
```

Comment:
We shouldn't have any indentation here since this is an independent block of code. Such spacing is necessary when we have nested blocks of code.

Reply:
Sure. Working on this. 👍

Reply:
LGTM